index.js

```diff
@@ -54,7 +87,17 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+// 1. loop through each imdbrating 
+// 2. convert or average out imbd rating
+// 3. return all results
+function getAverageIMDBRating(movies) {
+let meanRating = 0;
+
+  for(movie of movies){
+    meanRating = meanRating + Number(movie.imdbRating) / movies.length;
+  }
+  return meanRating;
```

Comment:
Nice work!  But currently we are dividing `meanRating` once for every single iteration and we could add up all of our ratings and only divide at the end.